test/index.ts

```diff
@@ -572,6 +580,34 @@ describe('BigQuery', () => {
       assert.strictEqual(timestamp.value, EXPECTED_VALUE);
     });
   });
+  // *************
+  // ********
+  // *****
+  // ***
+  // *
+  describe('geography', () => {
+    const INPUT_STRING = 'POINT(1 2)';
+
+    it.skip('should expose static and instance constructors', () => {
```

Comment:
Why skipped?

Reply:
It's skipped in the similar tests that I used as templates, so I left mine skipped. None of the custom data types' 'should expose static and instance constructors' tests appear to pass when un-skipped. Thoughts?

Reply:
I am 👎 on having skipped tests anywhere in the code base.  For the ones that were already there ...  I kinda don't know how they got there.   We shouldn't add new skipped ones for sure.  @callmehiphop can you shed some light here?

Reply:
@JustinBeckwith I can't, but git blame brought me here https://github.com/googleapis/nodejs-bigquery/pull/227/files#r228039589, so it looks like maybe we meant to deprecate those methods and never did?

Reply:
We should fire that guy

Reply:
Is there something I should change about the way I implemented geo, or am I able to just remove this test for my PR to be ok before we update the rest?

Reply:
I think we should just delete the skipped tests.

Reply:
I think we should remove that `skip` and see if we can get those tests passing, then open an issue to either remove/fix the other tests in a separate PR